3scale/porta
# Fix JWT Policy fields

**What this PR does / why we need it**:

This update supports new json schema options that are required by some policies (I.E: JWT)

**Which issue(s) this PR fixes** 

Fixes https://issues.jboss.org/browse/THREESCALE-3509

**Verification steps** 
* Chose a Product/API
* Visit Integration -> Policies
* Add JWT policy and configure its options.

It should work as expected

**Special notes for your reviewer**:
